common/djangoapps/student/tasks.py

```diff
@@ -0,0 +1,126 @@
+    """
+        Retrieve information about course for sending course enrollment email
+    """
+
+    course_run_fields = [
+        'key', 'title', 'short_description', 'marketing_url', 'pacing_type', 'min_effort',
+        'max_effort', 'weeks_to_complete', 'enrollment_count', 'image', 'staff',
+    ]
+
+    try:
+        user = User.objects.get(id=user_id)
+    except User.DoesNotExist:
+        user = None
+
+    try:
+        course_key = CourseKey.from_string(course_id)
+    except InvalidKeyError:
+        course_key = None
+
+    canvas_entry_properties = {
+        'instructors': [],
+        'course_date_blocks': [],
+        'learners_count': '',
+        'partner_image_url': '',
+        'banner_image_url': '',
+        'course_title': course_title,
+        'short_description': short_description,
+        'pacing_type': pacing_type,
+    }
+
+    owners, course_dates_list, course_date_blocks, course_run = [], [], [], None
+
+    try:
+        course_dates_list = get_course_dates_for_email(user, course_key, request=None)
+        course_date_blocks = list(map(_remove_date_key_from_course_dates, course_dates_list))
+    except Exception:  # pylint: disable=broad-except
+        pass
+
+    try:
+        course_uuid = get_course_uuid_for_course(course_id)
+        owners = get_owners_for_course(course_uuid=course_uuid)
+        course_run = get_course_run_details(course_id, course_run_fields)
+    except Exception:  # pylint: disable=broad-except
+        log.exception(
+            f'Unable to get data for user {user_id} from discovery for course {course_id}'
+        )
+
+    if course_run:
+        marketing_root_url = settings.MKTG_URLS.get('ROOT')
+        instructors = get_instructors(course_run, marketing_root_url)
+        enrollment_count = int(course_run.get('enrollment_count'))
+        canvas_entry_properties.update({
+            'instructors': instructors,
+            'instructors_count': 'even' if len(instructors) % 2 == 0 else 'odd',
+            'min_effort': course_run.get('min_effort'),
+            'max_effort': course_run.get('max_effort'),
+            'weeks_to_complete': course_run.get('weeks_to_complete'),
+            'learners_count': '{:,}'.format(enrollment_count) if enrollment_count > 100 else '',
+            'banner_image_url': course_run.get('image').get('src', '') if course_run.get('image') else '',
+            'course_title': course_run.get('title'),
+            'short_description': course_run.get('short_description'),
+            'pacing_type': course_run.get('pacing_type'),
+        })
+
+    price = CourseMode.min_course_price_for_currency(course_id=course_id, currency='USD')
+    canvas_entry_properties.update({
+        'goals_enabled': ENABLE_COURSE_GOALS.is_enabled(course_key),
+        'course_date_blocks': course_date_blocks,
+        'partner_image_url': (owners[0].get('logo_image_url') or '') if owners else '',
+        'learner_name': user.profile.name,
```

Comment:
We can get this from [Braze personalization tags](https://www.braze.com/docs/user_guide/personalization_and_dynamic_content/liquid/supported_personalization_tags/) in the template i.e `{{${first_name}}}`